services/solr/Dockerfile

```diff
@@ -0,0 +1,65 @@
+ARG SOLR_VERSION=9.6
+FROM tugboatqa/solr:${SOLR_VERSION}
+ENV COMPOSER_DISCARD_CHANGES=true
+ENV COMPOSER_NO_INTERACTION=true
+ENV COMPOSER_ALLOW_SUPERUSER=true
```

Comment:
We don't need these COMPOSER env vars since there is no code checkout on this service.